roboplot/config.py

```diff
@@ -19,3 +19,11 @@
 # Debugging image paths
 debug_image_file_path = os.path.normpath(os.path.join(resources_dir, 'Challenge_2_Test_Images', 'HackspacePath_Sample3.png'))
 debug_output_folder = os.path.normpath(os.path.join(resources_dir, 'DebugImages'))
+
+
+# Camera constants
+X_PIXELS_TO_POINTS_SCALE = 0.2
+Y_PIXELS_TO_POINTS_SCALE = 0.2
```

Comment:
Would these be better named `x_pixels_to_millimetres_scale` and `y_pixels_to_millimetres_scale`?